site/docs/spark-ddl.md

```diff
@@ -182,6 +182,28 @@ ALTER TABLE prod.db.sample
 ADD COLUMN point.z double
 ```
 
+```sql
+-- create a nested array column of struct
+ALTER TABLE prod.db.sample
+ADD COLUMN points array<struct<x: double, y: double>>;
+
+-- add a field to the struct with in array
+ALTER TABLE prod.db.sample
+ADD COLUMN points.element.z double
+```
+
+```sql
+-- create a map column of struct key and struct value
+ALTER TABLE prod.db.sample
+ADD COLUMN points map<struct<x: int>, struct<a: int>>;
+
+-- add a field to the value struct in a map
+ALTER TABLE prod.db.sample
+ADD COLUMN points.value.b int
```

Comment:
Nit: The comment is potentially ambiguous to me.

I would try to be more clear about the fact that `value` is a keyword for accessing a map element. (vs it just being possibly part of the existing table's definition). Again, the comment in the added test is great!